skiko/build.gradle.kts

```diff
@@ -221,10 +220,9 @@ val Project.supportAndroid: Boolean
     get() = findProperty("skiko.android.enabled") == "true" // || isInIdea
 
 kotlin {
+    jvmToolchain(17)
```

Comment:
I bumped the Java toolchain version to 17 because there was code around that would be marked red in the IDE since it was using newer APIs and the Gradle plugin wasn't able to pick up the correct language level (it would always reset to 6 when syncing).

But no there is no hard requirement on 17 — it should be 11, as I haven't changed anything else. I'm fixing this, sorry. Must have been when I was testing something out and I forgot to switch it back to 11.

Done in 7daf8b2fd72615467be16101d9661c4a592dd68a